server/src/main/scala/com/cloudera/livy/server/interactive/InteractiveSession.scala

```diff
@@ -79,7 +79,10 @@ class InteractiveSession(
         val pySparkFiles = if (!LivyConf.TEST_MODE) findPySparkArchives() else Nil
         builder.setConf(SparkYarnIsPython, "true")
         builder.setConf(SparkSubmitPyFiles, (pySparkFiles ++ request.pyFiles).mkString(","))
-
+      case SparkR() =>
+        val packageFile = livyConf.get("livy.repl.sparkr.package")
+        require(packageFile != null, "Can not find SparkR pakcage path.")
```

Comment:
"package". Also, why not look it up under "SPARK_HOME" if it's not explicitly set?

Generally, these configs exist so that the user can cache these files in HDFS, for example, making distribution quicker. But you should be able to find them without the user having to explicitly tell them where they are.